osmosis-1/osmosis-1.assetlist.json

```diff
@@ -9814,6 +9814,70 @@
         "osmosis-price:uosmo:1145"
       ]
     },
+    {
```

Comment:
the Assetlist should not be edited directly. Any changes of data should be done at the Chain Registry, where the automated script pulls this data from

Reply:
All added assets should go into the zone_assets file

Reply:
updated